api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityServiceTest.java

```diff
@@ -16,63 +25,140 @@
 import static uk.gov.dhsc.htbhf.claimant.testsupport.EligibilityResponseTestDataFactory.anEligibilityResponse;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleVoucherEntitlementTestDataFactory.aPaymentCycleVoucherEntitlementWithVouchers;
+import static uk.gov.dhsc.htbhf.claimant.testsupport.PaymentCycleVoucherEntitlementTestDataFactory.aPaymentCycleVoucherEntitlementWithZeroVouchers;
 import static uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus.DUPLICATE;
 import static uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus.ELIGIBLE;
+import static uk.gov.dhsc.htbhf.eligibility.model.EligibilityStatus.INELIGIBLE;
 
 @ExtendWith(MockitoExtension.class)
 class EligibilityServiceTest {
 
     @InjectMocks
-    private EligibilityService eligibilityService;
+    EligibilityService eligibilityService;
+
+    @Mock
+    EligibilityClient client;
 
     @Mock
-    private EligibilityClient client;
+    EligibilityStatusCalculator eligibilityStatusCalculator;
 
     @Mock
-    private EligibilityStatusCalculator eligibilityStatusCalculator;
+    ClaimRepository claimRepository;
 
     @Mock
-    private ClaimRepository claimRepository;
+    CycleEntitlementCalculator cycleEntitlementCalculator;
 
     @Test
     void shouldReturnDuplicateResponseWhenLiveClaimAlreadyExists() {
         given(claimRepository.liveClaimExistsForNino(any())).willReturn(true);
         Claimant claimant = aValidClaimant();
 
-        EligibilityResponse eligibilityResponse = eligibilityService.determineEligibilityForNewClaimant(claimant);
+        EligibilityAndEntitlement eligibilityResponse = eligibilityService.determineEligibilityForNewClaimant(claimant);
 
         assertThat(eligibilityResponse.getEligibilityStatus()).isEqualTo(DUPLICATE);
         verify(claimRepository).liveClaimExistsForNino(claimant.getNino());
-        verifyZeroInteractions(client, eligibilityStatusCalculator);
+        verifyZeroInteractions(client, eligibilityStatusCalculator, cycleEntitlementCalculator);
+    }
+
+    @Test
+    void shouldReturnEligibleWhenNoDuplicateAndEligibleWithVouchers() {
+        Claimant claimant = aValidClaimant();
+        EligibilityResponse eligibilityResponse = anEligibilityResponse();
+        PaymentCycleVoucherEntitlement voucherEntitlement = aPaymentCycleVoucherEntitlementWithVouchers();
+        given(claimRepository.liveClaimExistsForNino(any())).willReturn(false);
+        given(client.checkEligibility(any())).willReturn(eligibilityResponse);
+        given(eligibilityStatusCalculator.determineEligibilityStatusForNewClaim(any())).willReturn(ELIGIBLE);
+        given(cycleEntitlementCalculator.calculateEntitlement(any(), any(), any())).willReturn(voucherEntitlement);
+
+        EligibilityAndEntitlement result = eligibilityService.determineEligibilityForNewClaimant(claimant);
+
+        assertCorrectResponse(result, eligibilityResponse.getEligibilityStatus(), eligibilityResponse, voucherEntitlement);
```

Comment:
I think you should be specific about checking the status here by passing in `ELIGIBLE`.

Reply:
and have 5 parameters? How about `assertCorrectEligibleResponse`?